native/src/main/java/io/ballerina/stdlib/graphql/runtime/schema/FieldFinder.java

```diff
@@ -232,8 +258,10 @@ private void getFieldsFromInputObjectType(SchemaType schemaType, Field field) {
     private void getFieldsFromServiceType(SchemaType schemaType) {
         ServiceType serviceType = (ServiceType) schemaType.getBalType();
         for (ResourceMethodType resourceMethod : serviceType.getResourceMethods()) {
-            String[] resourcePath = resourceMethod.getResourcePath();
-            schemaType.addField(getFieldsFromResourceMethodType(resourceMethod, resourcePath));
+            if (Objects.equals(resourceMethod.getAccessor(), GET_ACCESSOR)) {
```

Comment:
This check is redundant as this method is used to get the fields of a service-type object. They all have the `get` accessor in the resource methods. (Only the top-level GraphQL service can have the `subscribe` accessor).